scripts/oneoff/fix_accidentially_published_g10_services.sql

```diff
@@ -0,0 +1,84 @@
+/*
+To apply this script run the below command
```

Comment:
Would perhaps be good to link here to the PR that fixes the broken thing that cause this to be necessary?

Reply:
Yep, will do when it's up!

Reply:
It's done!